MAINTAINING.md

```diff
@@ -0,0 +1,83 @@
+## Adding a new model
```

Comment:
Maybe note that this if for adding or updating a new "official" model. Any models that users train don't need to be registered in this way to be used. That way users aren't confused by these instructions and think they need to publish their models. In fact, it may be useful for users (as opposed to "maintainers") to have a link here that points to where they can distribute their models: https://github.com/drivendataorg/zamba/wiki